spec/TodoList.spec.js

```diff
@@ -1,144 +1,103 @@
-const TodoList = require("../src/TodoList.js")
+const TodoList = require('../src/TodoList.js')
 
-describe("TodoList", () => {
+describe('TodoList', () => {
   let todoList
 
   beforeEach(() => {
     todoList = new TodoList()
   })
 
-  it("creates a todo item", () => {
-    // set up
-    const expected = {
-      id: 1,
-      text: "turn the heating on!",
-      status: "incomplete"
-    }
-
-    // execute
-    const result = todoList.create("turn the heating on!")
-
-    // verify
-    expect(result).toEqual(expected)
+  // create(str)
+  it('creates a todo item', () => {
+    const result = todoList.create('turn the heating on!')
+    const expected = 'turn the heating on!'
+    expect(result.text).toEqual(expected)
```

Comment:
The simplification is nice on the surface but by doing this you no longer have test coverage of the ID and status being set as intended